media/css/firefox/onboarding/firstrun-fxa.less

```diff
@@ -3,18 +3,7 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 @import "../../sandstone/lib.less";
-
-.buildstar(@size, @left, @top, @speed, @delay) {
-    .animation(twinkle @speed ease-in-out @delay infinite alternate-reverse);
-    background: #fff;
-    border-radius: 50%;
-    height: @size;
-    left: @left;
-    opacity: 0;
-    position: absolute;
-    top: @top;
-    width: @size;
-}
+@import "onboarding-lib.less";
```

Comment:
You are totally right, `@keyframes` does generate CSS. My brain pattern matched it as a variable due to the `@`. I'm going to move `@keyframes twinkle` into `onboarding-common.less` and `firstrun-fxa.less` to keep with convention. This does result in some non-DRYness, but I think that's okay for now. When a winner/winners are chosen from this test, we'll refactor.